src/arg.rs

```diff
@@ -0,0 +1,64 @@
+use clap::{Arg, Command};
+
+const APP: &str = "Directory cleaner";
+pub struct Args {
+    pub types: Vec<String>,
+    pub min_size: u64,
```

Comment:
what does min size do?

Reply:
Checks to only deletes files with required size